distribution/dashboard/pkg/errors/error.go

```diff
@@ -0,0 +1,68 @@
+package errors
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type oberror struct {
```

Comment:
The error is actually HTTP status, call it oberror is misleading

Reply:
OK, it's true. I have renamed it to `buzerror`.